articles/native-platforms/ios-objc/10-customizing-lock.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Customizing Lock
+description: This tutorial will show you how to customize the Lock widget UI.
+---
+
+::: panel-info System Requirements
+This tutorial and seed project have been tested with the following:
+
+* CocoaPods 1.0.0
+* XCode 7.3 (7D175)
+* Simulator - iPhone 6 - iOS 9.3 (13E230)
+  :::
+
+<%= include('../../_includes/_package', {
+  pkgRepo: 'native-mobile-samples',
+  pkgBranch: 'master',
+  pkgPath: 'iOS/basic-sample-objc',
+  pkgFilePath: 'iOS/basic-sample-swift/SwiftSample/Info.plist',
+  pkgType: 'replace'
+}) %>
+
+### Before Starting
+
+You need the [Lock](https://github.com/auth0/Lock.iOS-OSX) library integrated in your project. Make sure you've followed the [login tutorial](01-login.md) and you know how to present the login dialog.
+
+### 1. Register your Theme
+
+Lock UI can be customized by creating your own `A0Theme` and overriding the default one anywhere before presenting the login dialog:
+
+```swift
+#import <Lock/Lock.h>
+```
+
+```swift
+A0Theme *myAwesomeTheme = [[A0Theme alloc] init];
+// Configure your theme here (see step 2)
+[[A0Theme sharedInstance] registerTheme: myAwesomeTheme];
+```
+
+### 2. Configure your Theme
+
+Lock's widget UI is composed of several parts that can be customized. Here is a list of the available values that you can modify:
+
+![Lock.png](/media/articles/libraries/lock-ios/customization/Lock-UI-Parts.png)
```

Comment:
@pablolvillar Please resize this image to max 750 pixels wide.

Also, according to our [guidelines](https://github.com/auth0/docs/blob/master/CONTRIBUTING.md#borders) a border should be added: _For close-ups and other screenshots that do not include the browser window, apply a 1px centered border (select all > edit > stroke) of color `#cccccc` to keep the image from blending with the background and appearing to float._